packages/react-native-editor/ios/GutenbergDemo.xcodeproj/project.pbxproj

```diff
@@ -706,11 +706,7 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
 				MTL_ENABLE_DEBUG_INFO = YES;
 				ONLY_ACTIVE_ARCH = YES;
-				OTHER_LDFLAGS = (
-					"$(inherited)",
-					"-Wl",
-					"-ld_classic",
-				);
+				OTHER_LDFLAGS = "$(inherited)";
```

Comment:
Do we know why this change is showing up for the release? It looks like it's also there in the Gutenberg Mobile PR.

Reply:
I see some of the changes that happened in [this PR](https://github.com/WordPress/gutenberg/pull/57667/files#diff-cc39ff1a662519f43783901ea3e7480ae1f48ec63dcaeded03149538471e8d22) are being reverted, but I'm not sure why.

I added the removed lines back locally then ran `bundle exec pod install --repo-update`, and they were removed again with the following warning:

```
[!] The `GutenbergDemo [Debug]` target overrides the `LIBRARY_SEARCH_PATHS` build setting defined in `Pods/Target Support Files/Pods-GutenbergDemo/Pods-GutenbergDemo.debug.xcconfig'. This can lead to problems with the CocoaPods installation
    - Use the `$(inherited)` flag, or
    - Remove the build settings from the target.

[!] The `GutenbergDemo [Release]` target overrides the `LIBRARY_SEARCH_PATHS` build setting defined in `Pods/Target Support Files/Pods-GutenbergDemo/Pods-GutenbergDemo.release.xcconfig'. This can lead to problems with the CocoaPods installation
    - Use the `$(inherited)` flag, or
    - Remove the build settings from the target.
```
So, it seems like these changes may be correct. @fluiddot, does this seems right to you, as the author of the original changes?

Reply:
The changes in the Xcode project are related to [this commit](https://github.com/facebook/react-native/commit/287482e57ffd221227e6fffb6852113d330260a1#diff-6626770c5918d7dc1e21b831d72ce4ff25ec0be572c60a4bf60dcb3aec20f80f) from React Native. AFAIK it's a patch for Xcode 15.

Reply:
@SiobhyB Not sure why they are getting reverted, I can try to install the pods locally and see if I get the same modifications.

Reply:
@SiobhyB Are you using Xcode 15? If not, maybe that could be the reason why they are being removed?

Reply:
Huh, I'm on Xcode 15.2. 🤔 Are they not removed when you run `bundle exec pod install --repo-update` locally?

Reply:
Actually, when I ran `xcodebuild -version` in the command line, I get `13.4.1`. I'll update now...

Reply:
I'm running `bundle exec pod install --repo-update` locally and the Xcode project remains unmodified. @SiobhyB I wonder what's the difference in our setup. Could you check the React Native version installed in your local environment?

You can quickly check the version by running `cat node_modules/react-native/package.json | jq '.version'` in Gutenberg.

Reply:
>Actually, when I ran xcodebuild -version in the command line, I get 13.4.1. I'll update now...

That might be the issue, yeah try changing the Xcode selection to use `15.2`

I've run `bundle exec pod install --repo-update` and I don't see any changes locally.

Reply:
I ran `sudo xcode-select --switch /Applications/Xcode.app` to update locally, then `bundle exec pod install --repo-update` produced the correct results. I've gone ahead to update in https://github.com/WordPress/gutenberg/pull/59461/commits/971081f2c0b819c4bd6de4adebf0d94f123cc979. I'll update Gutenberg Mobile now, too.

Thank you @geriux and @fluiddot! 🙌